src/Page/Explore/Impacts.elm

```diff
@@ -14,94 +14,114 @@ import Views.Impact as ImpactView
-                                ]
-                            , span [ class "d-flex flex-column" ]
-                                [ text "Pondération"
-                                , pefData.weighting |> Format.ratio
-                                ]
-                            ]
-
-                    Nothing ->
-                        text "N/A"
-      }
-    , { label = "Données de calcul du score d'impacts"
-      , toCell =
-            \def ->
-                case def.ecoscoreData of
-                    Just ecoscoreData ->
-                        div [ class "d-flex gap-2" ]
-                            [ span [ class "d-flex flex-column" ]
-                                [ text "Normalisation"
-                                , ecoscoreData.normalization |> Unit.impactToFloat |> Format.formatRichFloat 2 def.unit
-                                ]
-                            , span [ class "d-flex flex-column" ]
-                                [ text "Pondération"
-                                , ecoscoreData.weighting |> Format.ratio
-                                ]
-                            ]
-
-                    Nothing ->
-                        text "N/A"
-      }
-    , { label = "Niveau de qualité"
-      , toCell =
-            \def ->
-                def.quality
-                    |> ImpactView.impactQuality
-                    |> div [ classList [ ( "text-center", not detailed ) ] ]
-      }
-    , { label = "Source"
-      , toCell =
-            \def ->
-                a
-                    [ href def.source.url
-                    , target "_blank"
+        }
+            :: { label = "Nom"
+               , toValue = .label
+               , toCell =
+                    \def ->
+                        span [ title def.label ] [ text def.label ]
+               }
+            :: { label = "Unité"
+               , toValue = .unit
+               , toCell = \def -> code [] [ text def.unit ]
+               }
+            :: { label = "Normalisation (PEF)"
+               , toValue = .pefData >> Maybe.map (.normalization >> Unit.impactToFloat >> Format.formatFloat 2) >> Maybe.withDefault "N/A"
+               , toCell =
+                    \def ->
+                        def.pefData
+                            |> Maybe.map (.normalization >> Unit.impactToFloat >> Format.formatRichFloat 2 def.unit)
+                            |> Maybe.withDefault (text "N/A")
+               }
+            :: { label = "Pondération (PEF)"
+               , toValue = .pefData >> Maybe.map (.weighting >> Unit.ratioToFloat >> Format.formatFloat 2) >> Maybe.withDefault "N/A"
+               , toCell = .pefData >> Maybe.map (.weighting >> Format.ratio) >> Maybe.withDefault (text "N/A")
+               }
+            :: (if scope == Scope.Food then
+                    -- No "scope d'impacts" for textile
```

Comment:
My previous comment disappeared, so here it is again: if the intent of this refactor is to handle conditional list entries, we could use `List.filterMap` here. Otherwise, I feel like I'm missing the use case of chaining many `::` operator calls instead of using a plain old regular list…

Reply:
Yeah the `::` is the solution i went with as an elm-review rule wasn't happy about using `++` between lists which felt a bit nicer to me, but whatever. I could indeed use `Maybe`s  and `List.filterMap`, I guess i'll leave it as is for now.